src/Toolkit/Toolkit.Maui/BasemapGallery/BasemapGallery.Appearance.cs

```diff
@@ -48,97 +48,143 @@ static BasemapGallery()
-#else
-            Border border = new Border { Padding = new Thickness(0), StrokeThickness = 1, StrokeShape = new Rectangle() };
+        {
+            // Special template to take advantage of negative margin support on iOS, Mac
+
+            Border border = new Border
+            {
+                Padding = new Thickness(0),
+#if __IOS__
+                StrokeThickness = 8,
+#else
+                StrokeThickness = 1,
+#endif
+                StrokeShape = new Rectangle(),
+            };
             Grid outerScrimContainer = new Grid();
             outerScrimContainer.ColumnDefinitions.Add(new ColumnDefinition { Width = new GridLength(64) });
             outerScrimContainer.ColumnDefinitions.Add(new ColumnDefinition { Width = new GridLength(8) });
             outerScrimContainer.ColumnDefinitions.Add(new ColumnDefinition { Width = GridLength.Star });
             outerScrimContainer.ColumnSpacing = 0;
+#if __IOS__
+            outerScrimContainer.Margin = new Thickness(-4, -8, -8, -8);
+            outerScrimContainer.SetBinding(BackgroundColorProperty, static (Border border) => border.BackgroundColor);
+#endif
 
-            Image thumbnail = new Image { WidthRequest = 64, HeightRequest = 64, Aspect = Aspect.AspectFill, HorizontalOptions = LayoutOptions.Start };
-            Label nameLabel = new Label { FontSize = 12, HorizontalTextAlignment = TextAlignment.Start, VerticalTextAlignment = TextAlignment.Center };
-
+            Image thumbnail = new Image { WidthRequest = 64, HeightRequest = 64, Aspect = Aspect.AspectFill, HorizontalOptions = LayoutOptions.Start };
+#if __IOS__
+            thumbnail.SetBinding(BackgroundColorProperty, static (Border border) => border.BackgroundColor);
+#endif
+
             outerScrimContainer.Children.Add(thumbnail);
-            outerScrimContainer.Children.Add(nameLabel);
-
-            Grid.SetColumn(thumbnail, 0);
-            Grid.SetColumn(nameLabel, 2);
 
+            Grid.SetColumn(thumbnail, 0);
+
+            Grid newGrid = new Grid
+            {
+                Margin = new Thickness(0, 6, 6, 6)
+            };
+            newGrid.RowDefinitions.Add(new RowDefinition { Height = GridLength.Star });
+            newGrid.RowDefinitions.Add(new RowDefinition { Height = GridLength.Auto });
+
+            Label itemTextLabel = new Label
+            {
+                FontSize = 12,
+                HorizontalTextAlignment = TextAlignment.Start,
+                VerticalTextAlignment = TextAlignment.Center
+            };
+            itemTextLabel.SetBinding(Label.TextProperty, static (BasemapGalleryItem item) => item.Name, BindingMode.OneWay);
+            Grid.SetRow(itemTextLabel, 0);
+
+            Border itemTypeBorder = new Border
+            {
+                BackgroundColor = Application.Current?.RequestedTheme == Microsoft.Maui.ApplicationModel.AppTheme.Light ?
+                                    Colors.LightGray :
+                                    Colors.DarkGray,

```

Comment:
Don't hardcode colors based on current theme - these should be app theme bindings to they update if the mode changes. ie
`itemTypeBorder.SetAppThemeColor(BackgroundColorProperty, Colors.LightGray, Colors.DarkGray);`